docker/Makefile

```diff
@@ -1,61 +1,103 @@
 dc=docker-compose
 d=docker
-dcp=docker-compose -f docker-compose-production.yml
+dcb=$(dc) -f docker-compose-build.yml
+dcp=$(dc) -f docker-compose-production.yml
 
+.PHONY: migrate create-env build-static watch-static \
+build test backup restore \
+generate-production-static-data deploy
+
+
+# ---------------------- Dev section ----------------------
 migrate:
-	$(dc) exec stb-python python manage.py migrate
+	$(dc) exec app python manage.py migrate
+
+fixtures:
+	$(dc) run --rm app bash -c "\
+		python manage.py loaddata stroyprombeton/fixtures/admin.json \
+		&& python manage.py loaddata stroyprombeton/fixtures/dump.json \
+	"
+
+prices:
+	$(dc) exec app python manage.py price
 
-# for local env
-static:
-	@docker-compose run --rm stb-nodejs bash -c "\
-		npm install && \
-		npm install -g gulp-cli && \
-		gulp build"
+create-env:
+	@bash ./create-env.sh
+
+create-config:
+	@bash ./create-config.sh
+
+build-static:
+	$(dc) run --rm nodejs gulp build
+
+watch-static:
+	$(dc) run --rm nodejs gulp watch
 
-# for local env
 build: static
-	$(dc) build stb-python
+	$(dc) build app
+
+collectstatic:
+	$(dc) exec app python manage.py collectstatic --noinput
 
-# for local env
-test: static
-	$(dc) up -d stb-selenium
-	$(dc) exec stb-python python manage.py test --liveserver=stb-python:8001-8010 --parallel=8
+test: build-static
+	$(dc) up -d selenium
+	$(dc) exec app python manage.py test --liveserver=app:8001-8009 --parallel
 	$(dc) stop
 
-# for local env
 lint:
-	$(dc) up -d stb-python
-	$(d) run --rm --volumes-from stb-python --workdir=/usr/app/src -it coala/base coala
+	$(dc) up -d app
+	$(d) run --rm --volumes-from app --workdir=/usr/app/src -it coala/base coala
 	$(dc) stop
 
-# for local env
-restore:
-	@bash ../etc/stb-backup-restore.sh
 
-# for serv env
+# ---------------------- Deploy section ----------------------
+deploy-dev:
+	$(MAKE) create-env
+	$(MAKE) create-config
+	$(dc) pull
+	$(dc) up -d app
+	$(MAKE) build-static
+	# Create admin user with login/pass: admin/asdfjkl;
+	# Launch "collectstatic" not in static recipe because ManifestStaticStorage writes to db
+	$(dc) exec app bash -c "\
```

Comment:
Maybe we should reuse existing receipts instead of it, what do you think?
```
$(MAKE) migrate
$(MAKE) load-fixtures
$(MAKE) collectstatic
```

Reply:
TODO